NEWS.md

```diff
@@ -26,6 +26,899 @@ Refer to the Roadmap issue for complete list on on-going version features.
 If you check in something that is not reflected in Roadmap issue, please reply
 to that issue so it can get added.
 
+## 0.6
+
+### Relay in Production
+Relay is a functional, differentiable programming language designed to be an expressive intermediate representation for machine learning systems. Relay supports algebraic data types, closures, control flow, and recursion, allowing it to directly represent more complex models than computation graph-based IRs (e.g., NNVM) can. In TVM v0.6, Relay is in stable phase and is ready for production.
+
+* Algebraic Data Types (ADT) support ([#2442](#2442), [#2575](#2575)). ADT provides an expressive, efficient, and safe way to realize recursive computation (e.g., RNN). Refer to https://docs.tvm.ai/langref/relay_adt.html for more information.
```

Comment:
remove the bracket [#2442](#2442) because it will only link to a internal location of news which

Reply:
you mean sth like `Algebraic Data Types (ADT) support (#2442, #2575)` ?

Reply:
yes

Reply:
ping @yzhliu